### PR TITLE
Fix systemPreferences docs

### PR DESCRIPTION
**Pull request by miniak**

### CODE REVIEW

**Comment by danhp** [comment]
Not quite correct as `AppleShowScrollBars` is  a `string`

File: docs/api/system-preferences.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ are:
 * `AppleInterfaceStyle: string`
 * `AppleAquaColorVariant: integer`
 * `AppleHighlightColor: string`
-* `AppleShowScrollBars: string`
+* `AppleShowScrollBars: boolean`
```
Comment (zcbenz) [inline docs/api/system-preferences.md]: Even though it looks like a `boolean` it is actually a `string`.

**Comment by miniak** [comment]
I have updated the PR to only change the broken sample code

**Comment by zcbenz** [comment]
Thanks!

**Pull request merged**